### ISSUE TITLE
Sainlogic WS3500 - Listener - WU - illegal Request

### ISSUE THREAD

**Issue by anarky84666** [phifogg/ioBroker.sainlogic#67]
I set everything by the howto.
My Sainlogic WS3500 reaches ioBroker, but the arriving Data is not excepted:

(5457) Listener received illegal request: /weatherstation/updateweatherstation.php?ID=Wittekindstr1&PASSWORD=123qwe&indoortempf=77.9&tempf=73.9&dewptf=37.9&windchillf=73.9&indoorhumidity=33&humidity=2

Everything is up-to-date.

**Comment by anarky84666** [comment]
I solved my problem.
In the "WS View"-App the default-Path is 
/weatherstation/updateweatherstation.php?
In your UI the default is
/weatherstation/updateweatherstation.php

Your Tutorial say, I have to use the same on both sites. So I added a ? in your configuration-UI.
Their is an if-Statement in your Code, that force the Error, if both is not equal.

In your Code arrived  /weatherstation/updateweatherstation.php?, although in the App is configured something else.

How ever, with Default-Values in App and ioBroker-Conf, it works.

Sorry for the Bug.
